proc/proc_test.go

```diff
@@ -2357,3 +2357,20 @@ func TestNegativeIntEvaluation(t *testing.T) {
 		}
 	})
 }
+
+func TestIssue683(t *testing.T) {
+	// Step panics when source file can not be found
+	withTestProcess("issue683", t, func(p *Process, fixture protest.Fixture) {
+		_, err := setFunctionBreakpoint(p, "main.main")
+		assertNoError(err, t, "setFunctionBreakpoint()")
+		assertNoError(p.Continue(), t, "First Continue()")
+		for i := 0; i < 20; i++ {
+			// eventually an error about the source file not being found will be
+			// returned, the important thing is that we shouldn't panic
+			err := p.Step()
+			if err != nil {
```

Comment:
We should set some sort of state here that if we never get an error, the test should fail.